charts/enterprise-metrics/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 1.4.1
+version: 1.4.2
 appVersion: v1.4.1
```

Comment:
I think at the time the download for v1.4.2 didn't exist but it won't hurt to update the PR to include this. Note, that despite the versioning appearing to match up, the Helm chart versioning is completely separate to the GEM versioning. It's a bit too close at the moment that it causes confusion but I expect it will diverge at some point.